modules/proc_configuring-by-using-the-ocp-web-console.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * argocd_instance/argo-cd-cr-component-properties.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-the-notificationsconfiguration-by-using-the-web-console_{context}"]
+= Configuring the NotificationsConfiguration CR by using the web console
+
+You can configure the `NotificationsConfiguration` CR by using the web console.
+
+.Prerequisites
+
+* You have access to an {OCP} cluster with `cluster-admin` privileges and are logged into the web console.
+* You have installed the {gitops-title} Operator on your cluster.
+* You have enabled notifications for the Argo CD instance. For more information, see xref:../argocd_instance/argo-cd-cr-component-properties#gitops-argo-cd-notification_argo-cd-cr-component-properties[Enabling notifications with an Argo CD instance].
+
+.Procedure
+
+. Go to the *NotificationsConfiguration* page and click `default-notifications-configuration`.
+. On the *default-notifications-configuration* page, click *YAML* and add the configuration for any supported resources such as `templates`, `triggers`, `services`, and `subscriptions`. For example, under `templates` in the code, add the following sample configuration:
++
+.Example template configuration
+[source,yaml]
+----
+  template.my-custom-template: |
+    message: |
+    Application details: {{.context.argocdUrl}}/applications/{{.app.metadata.name}}.
+----
+
+. Click *Save*.
+
+. Verify the changes on the `NotificationsConfiguration` page by performing the following steps:
```

Comment:
Up to you but I think the end of this step could be removed and the meaning would still be clear:
```suggestion
. Verify the changes on the `NotificationsConfiguration` page:
```

Reply:
Fixed.